decidim-proposals/lib/decidim/proposals/feature.rb

```diff
@@ -37,6 +37,19 @@
     resource.template = "decidim/proposals/proposals/linked_proposals"
   end
 
+  feature.register_stat :accepted_proposals_count, primary: true do |features, start_at, end_at|
```

Comment:
This is primary because it is the big number displayed in the homepage. See the image in the PR's description.